yb-voyager/cmd/callhome_payload_builder.go

```diff
@@ -333,10 +333,10 @@ func getAnonymizedDDLs(sourceDBConf *srcdb.Source) []string {
 // ============================export schema callhome payload information============================
 
 const (
-	REDUNDANT_INDEX_CHANGE_TYPE               = "redundant_index"
-	TABLE_SHARDING_RECOMMENDATION_CHANGE_TYPE = "table_sharding_recommendation"
-	MVIEW_SHARDING_RECOMMENDATION_CHANGE_TYPE = "mview_sharding_recommendation"
-	SECONDARY_INDEX_TO_RANGE_CHANGE_TYPE      = "secondary_index_to_range"
+	REDUNDANT_INDEX_CHANGE_TYPE                 = "redundant_index"
+	TABLE_COLOCATION_RECOMMENDATION_CHANGE_TYPE = "table_colocation_recommendation"
+	MVIEW_COLOCATION_RECOMMENDATION_CHANGE_TYPE = "mview_colocation_recommendation"
+	SECONDARY_INDEX_TO_RANGE_CHANGE_TYPE        = "secondary_index_to_range"
```

Comment:
can this require bumping up payload version number?

Reply:
how are we using in callhome this change type?
If not significant maybe you can keep it consistent.